Behavioral/Command/Tv/Receiver.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace DesignPatterns\Behavioral\Command\Tv;
+
+class Receiver
+{
+    public $currentChannel = 0;
```

Comment:
Why is this public? By making this pulic, you can set the channel without calling `changeChannel()` and bypassing the logic in there :-1: